packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -108,6 +109,7 @@ export const createSDKConnectionValidator = z
     encryptPayload: z.boolean(),
     proxyEnabled: z.boolean().optional(),
     proxyHost: z.string().optional(),
+    sseEnabled: z.string().optional(),
```

Comment:
This is not a string. Also this shouldn’t be specified by the front end at all, so shouldn’t be part of the create/edit methods in the first place